arc_design_contest/2020/SEU_Intelligent_Recognition_System_For_Fall_Posture/src/NNLayer.c

```diff
@@ -0,0 +1,175 @@
+#include "NNLayer.h"
+#include "W0.h"
+#include "B0.h" 
+#include "W1.h"
+#include "B1.h"
+#include "math.h"
+//#include "W2.h"
+//#include "B2.h"
+//extern xdata NNLayer gMyCalculatorModel[LAYER_TOTAL];
+//extern xdata double gLayer0OutVal[LAYER_0_OUTDIM],gLayer1OutVal[LAYER_1_OUTDIM];
+
+extern NNLayer gMyCalculatorModel[LAYER_TOTAL];
+extern double gLayer0OutVal[LAYER_0_OUTDIM],gLayer1OutVal[LAYER_1_OUTDIM];//,gLayer2OutVal[LAYER_2_OUTDIM];
+  
+/*
+*   ѵ��õ�ģ�Ͳ���
+*/
+
+
+ 
+/*
+*   return arr[r][c]
+*/
+double _getWeight(const double* arr,int r,int rNum,int c)      
+{
+    return *(arr + r*rNum + c);
+}
+ 
+/*
+*   NNLayer��ʼ��
+*   ��ÿ���Ȩ�ؾ���ƫ�þ���
+*   ����ÿ�������ά�ȣ����ά��
+*   ��ÿ�������ݴ�����
```

Comment:
Please check the comments and translate to English

Reply:
Finish translating to English